Create project idea

ayan-biswas0412 opened the pull request:
## Describe your project in 1-3 sentences

I am building an intelligent door access control system which will have the following features 
  * Camera-based face recognition
  * RFID-based card lock system for user authentication.

## Hack Club Slack handle

[@AYAN BISWAS](https://hackclub.slack.com/team/U01584DM9PH)

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

bellesea commented on ayan-biswas0412.md:
```diff
@@ -0,0 +1,49 @@
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Raspberry Pi 3 - Model B - ARMv8 with 1G RAM   | [https://www.adafruit.com/product/3055](https://www.adafruit.com/product/3055) | $35  |
```
it looks like this is out of stock, do you have another source/site where you'd be able to purchase the pi?

bellesea replied:
also, do you need an SD card?

ayan-biswas0412 replied:
No I don't need an SD card, and I have Amazon as my alternative from where I will be purchasing if this goes out of stock in adafruit.

bellesea replied:
sounds good, thanks!

bellesea commented on ayan-biswas0412.md:
```diff
@@ -0,0 +1,49 @@
+    * Power supply
+2. Set up the Raspberry Pi and camera module according to the manufacturer's instructions.
+3. Connect the servo motor to the Raspberry Pi and wire it to control the lock mechanism.
+4. Write and upload code to the Raspberry Pi that will:
```
how do you plan on running the facial recognition algorithm?

ayan-biswas0412 replied:
I have my algorithm ready and it is part of my previous work which I will be using here.

bellesea commented on ayan-biswas0412.md:
```diff
@@ -0,0 +1,49 @@
+| Wires and connectors  | [https://www.adafruit.com/product/1957](https://www.adafruit.com/product/1957) | $1.95  |
+| 5V 2.5A Switching Power Supply with 20AWG MicroUSB Cable   | [https://www.adafruit.com/product/1995](https://www.adafruit.com/product/1995) | $8.25  |
+| HDMI 5" 800x480 Display Backpack - With Resistive Touchscreen   | [https://www.adafruit.com/product/2260](https://www.adafruit.com/product/2260) | $74.95  |
+| Speaker   | [https://www.adafruit.com/product/3885](https://www.adafruit.com/product/3885) | $5.95  |
```
what's the role of the speaker?

ayan-biswas0412 replied:
It is for making a sound like -> 
Access Granted/Denied for the user

bellesea replied:
ahh cool!!

bellesea commented on ayan-biswas0412.md:
```diff
@@ -0,0 +1,49 @@
+    * Wires and connectors
+    * Power supply
+2. Set up the Raspberry Pi and camera module according to the manufacturer's instructions.
+3. Connect the servo motor to the Raspberry Pi and wire it to control the lock mechanism.
```
do you plan on breadboarding this or soldering the components together? and, do you have the tools you need?

ayan-biswas0412 replied:
I plan to solder them together, and yes I have necessary soldering iron and other stuffs in my home.

bellesea replied:
great, thank you!

bellesea reviewed (changes requested):
@ayan-biswas0412 this is really really cool! i'm wondering, are you following any sort of guide for this?

and also left some specific comments hehe :)

thank you!

ayan-biswas0412 commented:
> @ayan-biswas0412 this is really really cool! i'm wondering, are you following any sort of guide for this?
> 
> and also left some specific comments hehe :)
> 
> thank you!

Hi @bellesea thanks for reviewing my file, I have addressed all of your comments, can you please check the previous conversations?

And yes, I got motivation for the project from some youtube videos and online tutorials.

bellesea reviewed (approved):
@ayan-biswas0412 this looks awesome! can't wait to see this being worked on hehe :)

an awesome member of @hackclub/bank-ops will be reaching out next week with next steps!

btw, could you please add to the file and credit some resources you're referencing? thank you :)

ayan-biswas0412 commented:
> @ayan-biswas0412 this looks awesome! can't wait to see this being worked on hehe :)
> 
> an awesome member of @hackclub/bank-ops will be reaching out next week with next steps!
> 
> btw, could you please add to the file and credit some resources you're referencing? thank you :)

Yeah I have attached some references in my file. Thanks for approving the PR @bellesea

MelSmith104 reviewed (commented):
hey @ayan-biswas0412 this projects looks so cool! i love that you want to build it because of your interest in improving security - im really excited to see you make this! im dm'ing you on slack right now to chat about next steps!

ayan-biswas0412 commented:
Thanks @MelSmith104 for merging the PR.
